android/app/build.gradle

```diff
@@ -186,6 +186,8 @@ dependencies {
 
 }
 
+
```

Comment:
I guess this changes are done accidentally.

Reply:
Yeah I guess this was made accidentally

Reply:
Please remove this change. :)